test/correctness/output_larger_than_two_gigs.cpp

```diff
@@ -11,29 +11,39 @@ void halide_error(void *ctx, const char *msg) {
 using namespace Halide;
 
 int main(int argc, char **argv) {
+    Target t = get_target_from_environment();
+
     Var x, y, z;
     Func identity_uint8;
     identity_uint8(x, y, z) = cast<uint8_t>(42);
-
-    uint8_t c[4096];
-    memset(c, 42, sizeof(c));
-
-    buffer_t buf;
-    memset(&buf, 0, sizeof(buf));
-    buf.host = c;
-    buf.extent[0] = 4096;
-    buf.extent[1] = 4096;
-    buf.extent[2] = 256;
-    buf.stride[0] = 1;
-    buf.stride[1] = 0;
-    buf.stride[2] = 0;
-    buf.elem_size = 1;
-
     identity_uint8.set_error_handler(&halide_error);
 
-    Buffer output_buf(UInt(8), &buf);
-    identity_uint8.realize(output_buf);
-
-    assert(error_occurred);
+    if (t.bits == 64) {
+        Image<uint8_t> output(4096, 4096, 256);
```

Comment:
This image is too large for our buildbots to run this test. They just fail to allocate it.